.mockery.yaml

```diff
@@ -46,6 +46,11 @@ packages:
       ClientWithResponsesInterface:
         config:
           filename: client.go
+  github.com/astronomer/astro-cli/runtime-template-client:
```

Comment:
Not used anymore?

Reply:
I have cleaned this unused code as we moved away from initial approach of creating top level client.